### PR TITLE
Add Citizen Web3 RPCs

### PR DESCRIPTION
**Pull request by alibabaedge**

### CODE REVIEW

File: celestia/chain.json [hunk 1]
```diff
@@ -235,6 +235,10 @@
         "address": "https://celestia-rpc.enigma-validator.com",
         "provider": "Enigma"
       },
+      {
```
Comment (reversesigh) [inline celestia/chain.json:238]: New entries needed to be added to the bottom of the list please.

File: composable/chain.json [hunk 2]
```diff
@@ -312,6 +312,10 @@
         "address": "https://rpc.centauri.bronbro.io:443",
         "provider": "Bro_n_Bro"
       },
+      {
```
Comment (reversesigh) [inline composable/chain.json:315]: New entries needed to be added to the bottom of the list please.

File: evmos/chain.json [hunk 3]
```diff
@@ -419,6 +419,10 @@
       {
         "address": "https://rpc.evmos.bronbro.io:443",
         "provider": "Bro_n_Bro"
+      },
+      {
```
Comment (reversesigh) [inline evmos/chain.json:423]: This one is fine

File: composable/chain.json (hunk 2 above)
Comment (alibabaedge) [reply composable/chain.json:315]: Fix it

**Pull request merged**